messagebird/signed_request.py

```diff
@@ -0,0 +1,42 @@
+import hashlib
+import hmac
+import base64
+import time
+from collections import OrderedDict
+
+try:
+    from urllib.parse import urlencode
+except ImportError:
+    from urllib import urlencode
+
+
+class SignedRequest:
+
+    def __init__(self, requestSignature, requestTimestamp, requestBody, requestParameters):
+        self._requestSignature = requestSignature
+        self._requestTimestamp = str(requestTimestamp)
+        self._requestBody = requestBody
+        self._requestParameters = requestParameters
+
+    def verify(self, signing_key):
+        payload = self._build_payload()
+        expected_signature = base64.b64decode(self._requestSignature)
+        calculated_signature = hmac.new(signing_key.encode('latin-1'), payload.encode('latin-1'), hashlib.sha256).digest()
+        return expected_signature == calculated_signature
+
+    def is_recent(self, offset = 10):
```

Comment:
what do you think about making 10 as constant?

Reply:
I don't think we should, it's just the default value doesn't have a meaning for is reusable